services/management/ia/common.go

```diff
@@ -35,6 +35,8 @@ func convertParamUnit(u string) iav1beta1.ParamUnit {
 	switch strings.ToLower(u) {
 	case "%", "percentage":
 		return iav1beta1.ParamUnit_PERCENTAGE
+	case "s", "sec", "second", "seconds":
+		return iav1beta1.ParamUnit_SECONDS
```

Comment:
Please do the same thing as in https://github.com/percona/pmm-managed/pull/590/file to avoid merge conflicts

Reply:
Done